api_tests/lib/ledger_runner.ts

```diff
@@ -71,7 +71,7 @@ export class LedgerRunner {
                 bitcoin.init(await this.getBitcoinClientConfig());
                 this.blockTimers.bitcoin = global.setInterval(async () => {
                     await bitcoin.generate();
-                }, 1000);
+                }, 2000);
```

Comment:
This is only needed to not miss blocks right? 
Can you add a note to the "Get blocks from the past" ticket to reduce this amount again please?